src/vst2/Vst2PluginInstance.h

```diff
@@ -28,6 +28,7 @@ class Vst2PluginInstance : public AudioEffectX
    virtual VstInt32 processEvents(VstEvents* ev);
 
    // Host->Client calls
+   virtual bool Vst2PluginInstance::getProgramNameIndexed(VstInt32 category, VstInt32 index, char* text);
```

Comment:
While it's valid to fully qualify the name here (Vst2PluginIntance::getProgramNameIndexed) you don't need to and we generally don't I think you can remove the qualifier.